internal/logger/logger.go

```diff
@@ -0,0 +1,46 @@
+package logger
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"go.uber.org/zap"
+)
+
+var Log *zap.Logger = zap.NewNop()
+
+func Initialize(level string) error {
+	// преобразуем текстовый уровень логирования в zap.AtomicLevel
+	lvl, err := zap.ParseAtomicLevel(level)
+	if err != nil {
+		return fmt.Errorf("%w", err)
```

Comment:
Добавляй ошибкам контекст

```suggestion
		return fmt.Errorf("failed to parse the atomic level %s: %w", level, err)
```

Reply:
done